index.html

```diff
@@ -0,0 +1,751 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="keywords" content="CSS, styles, styleSUP,Supminn,SupTECH, Web developer">
+    <meta name="description" content="styleSUP CSS Component Library">
+    <meta name="author" content="Supriya M">
+    <meta name="Copyright" content="SupTECH">
+    <link rel="icon" href="./images/logo.png" type="image/png">
+    <link rel="stylesheet" href="./styles/components.css">
+    <link rel="stylesheet" href="./styles/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css">
+    <title>StyleSUP | Docs</title>
+</head>
+
+<body>
+    <div class="grid-body">
+        <!-- Navigation -->
+        <nav id="navigation" class="nav">
+            <div class="nav-logo">
+                <img class="img-rd" src="./images/logo.png" alt="logo" />
+                <span class="nav-header">StyleSUP</span>
+            </div>
+            <div class="social-media">
+                <a class="link" target="_blank" href="https://github.com/supminn">
+                    <img class="media-logo" src="./images/github.svg" alt="github" />
+                </a>
+                <a class="link" target="_blank" href="https://twitter.com/supminn">
+                    <img class="media-logo" src="./images/twitter.svg" alt="twitter" />
+                </a>
+            </div>
```

Comment:
Flex would be better option here I guess because it is one dimensional.

Reply:
I agree, not sure why I used grid. Will get it changed. Thanks!